module.go

```diff
@@ -18,82 +15,61 @@ import (
 	"github.com/signalsciences/tlstext"
 )
 
-const moduleVersion = "sigsci-module-golang " + version
-
 // Module is an http.Handler that wraps an existing handler with
 // data collection and sends it to the Signal Sciences Agent for
 // inspection.
 type Module struct {
-	handler                   http.Handler
-	rpcNetwork                string
-	rpcAddress                string
-	debug                     bool
-	timeout                   time.Duration
-	anomalySize               int64
-	anomalyDuration           time.Duration
-	maxContentLength          int64
-	allowUnknownContentLength bool
-	moduleVersion             string
-	serverVersion             string
-	inspector                 Inspector
-	inspInit                  InspectorInitFunc
-	inspFini                  InspectorFiniFunc
-	headerExtractor           func(*http.Request) (http.Header, error)
+	config          *ModuleConfig
+	handler         http.Handler
+	inspector       Inspector
+	inspInit        InspectorInitFunc
+	inspFini        InspectorFiniFunc
+	headerExtractor HeaderExtractorFunc
```

Comment:
Isnt headerExtractor already in the config object(ModuleConfig). Repeated here? Same for inspFini, et al?

Reply:
Yes, but it is done this way as it is a bit more efficient.